config/environments/development.rb

```diff
@@ -16,6 +16,8 @@
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = false
 
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.